cmake/run_test.cmake

```diff
@@ -62,6 +62,15 @@ if(output_blessed)
     )
 
   if( test_not_successful )
+    if(1)
```

Comment:
What's the if(1) for?

Reply:
for some test for the existence of the diff command, not sure what is portable enough here

Reply:
sure, but why the if(1) ?

Reply:
just a placeholder ...

Reply:
I've now implemented detection for diff